style.css

```diff
@@ -227,10 +253,12 @@ p {
 .dropdown {
   display: flex;
   flex-direction: column;
-  padding: 0 36px;
+  max-width: 70%;
+  padding: 0;
+  margin-bottom: 57px;
 }
```

Comment:
- [ ] Kindly make sure the `max-width: 70%` is only taking effect on the desktop version but not on the mobile version. I noticed that the mobile version has changed. Please fix that to meet the provided Figma design.

   - ### YOUR DESIGN
  
    ![image](https://user-images.githubusercontent.com/87197412/213569200-d8ef01d2-490c-4063-aa31-4324a6d2ee2e.png)

   - ### Expected design
   
    ![image](https://user-images.githubusercontent.com/87197412/213569307-db996e11-5cb1-4148-89ae-70e7baee0402.png)